modules/shared/components/molecules/UploadingImage/UploadingImage.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import type { Story, Meta } from '@storybook/react';
+import UploadingImage from './UploadingImage';
+import type { IUploadingImage } from './IUploadingImage';
+
+export default {
+  title: 'molecules/UploadingImage',
+  component: UploadingImage,
+  argTypes: {},
+} as Meta;
+
+const Template: Story<IUploadingImage.IProps> = (args) => (
+  <UploadingImage {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  imageUrl: 'https://source.unsplash.com/random',
```

Comment:
shouldn't we check the base64 url instead?

Reply:
the base 64 logic is not ready yet, hence I used an imageUrl as a prop so it will receive whatever the text passed whether it was a base 64 or urlimage from API